docs/contributor/03-11-hap-rules.md

```diff
@@ -0,0 +1,244 @@
+
+The following table summarizes the attributes that can be used in the rule entry. Its columns define:
+* Name and symbol - the latter is used in rule entries.
+* Data type and possible values.
+* Input attributes - true when the attribute can occur in the input attributes section (left part) of a rule entry (see [Rule Format](#rule-format) section).
+* Output attributes - true when the attribute can occur in the output attributes section (right part) of a rule entry (see [Rule Format](#rule-format) section).
+* Modified SecretBinding search labels - lists modifications of search labels that the rule entry containing the attribute adds to search labels if triggered. 
+
+| Name (Symbol)        	| Data Type and Possible Values                                                                                                                    	| Input Attribute 	| Output Attribute 	| Modified SecretBinding Search Labels                                                              	|
+|----------------------	|------------------------------------------------------------------------------------------------------------------------------------	|-----------------	|------------------	|---------------------------------------------------------------------------------------------------	|
+| Platform Region (PR) 	| string, platform regions as defined in https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment 	| true            	| false            	| hyperscalerType: `<providerType>_<PR>` or hyperscalerType: `<providerType>_<PR>_<HR>` if used with HR 	|
+| Hyperscaler Region (HR)  	| string, hyperscaler region as defined in https://help.sap.com/docs/btp/sap-business-technology-platform/regions-for-kyma-environment   	| true            	| false            	| hyperscalerType: `<providerType>_<HR>` or hyperscalerType: `<providerType>_<PR>_<HR>` if used with PR 	|
+| Eu Access (EU)       	| no value                                                                                                                           	| false           	| true             	| euAccess: true                                                                                    	|
+| Shared (S)           	| no value                                                                                                                           	| false           	| true             	| shared: true                                                                                      	|
+
+
+
+## Validation
+
+KEB validates HAP Rules during startup. If the configuration is invalid, KEB will not start and an error message will be displayed in the logs. 
+
+The constraints used for validation during KEB startup include:
+* Rules format check - all the rules must comply with the format specified above.
+* Every rule entry must refer to an existing plan - if you define a rule with non existing plan then an error is returned during KEB startup that fails the process.
```

Comment:
Not sure about that rule, disabling preview plan will require removing hap.rule for preview.

Reply:
Removed.